README.md

```diff
@@ -84,6 +84,22 @@ or [Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-watson).
 ## Examples
 The [examples][examples] folder has basic and advanced examples.
 
+### Document Conversion integration with Retrieve and Rank
```

Comment:
Add this example below the `Document Conversion` example and make sure there is a link from the table of content.